packages/paste-style-props/src/proptypes/space.ts

```diff
@@ -4,7 +4,9 @@ import {propValidator} from './utils/propValidator';
 // Tokens
 const SpaceOptions = Object.keys(DefaultTheme.space);
 const MarginOptions = ['auto', ...Object.keys(DefaultTheme.space)];
+const GapOptions = Object.keys(DefaultTheme.space);
 
 export const isSpaceTokenProp = propValidator(SpaceOptions);
 export const isPaddingTokenProp = propValidator(SpaceOptions);
 export const isMarginTokenProp = propValidator(MarginOptions);
+export const isGapTokenProps = propValidator(GapOptions);
```

Comment:
nit: you could have done `export const isGapTokenProps = propValidator(SpaceOptions);` but no big deal